### DISCUSSION TITLE
[Projects Beta] Manage Access using Organization Teams

### DISCUSSION THREAD

**Discussion by jwfuller** [community/community#6756] [Projects and Issues] [answered]
I would like to assign Read, Write, Admin access to a project using Teams that are defined at the Organization level.

**Comment by obryckim** [comment]
> I would like to assign Read, Write, Admin access to a project using Teams that are defined at the Organization level.

I agree. We use teams exclusively and this would be a great feature! 👍🏻

**Comment by timoyates** [comment]
Thanks for the feedback, folks! This is on our roadmap and will indeed be possible. I don't have a timeline at this exact moment, but we'll add it to the [public roadmap](https://github.com/orgs/github/projects/4247/views/7) when we do. Cheers!

**Comment by azenMatt** [comment, marked as answer]
Hi @jwfuller @obryckim - this is now live so you can add a team under the manage access page in the same way you would add individuals 😄

**Reply by obryckim** [reply]
Is this rolling out in phases? I still only have the ability to add users (not teams) to the projects. Thanks!

**Reply by azenMatt** [reply]
hi @obryckim - can you try this again? Should work for you now...

**Reply by obryckim** [reply]
@azenMatt, thank you. I do have that ability now.
